pages/en/lb3/readmes/loopback-connector-ibmi.md

```diff
@@ -0,0 +1,136 @@
+- `loopback-connector-db2iseries` requires separate licensing to be fully functional. This module does not.
+- This package requires manual installation of prerequisites (see below), whereas `loopback-connector-db2iseries` can handle this automatically as part of the npm install
+
+## Prerequisites
+Before installing this package, you will need an ODBC driver and a driver manager (with development libraries). 
+This package is primarily developed and tested with the [IBM i Access ODBC driver](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/rzaik/rzaikappodbc.htm), which is supported as part of IBM i software maintenance agreements (SWMA) and comes with no additional licensing fees. 
+
+### On IBM i
+- Install the `unixODBC-devel` package. See [the RPM and yum documentation for IBM i](http://ibm.biz/ibmi-rpms) for more detailed steps.
+### On Linux
+- Install the `unixODBC-devel` package with your operating system's package manager (apt-get, zypper, yum, etc).
+- Install the "Linux Application Package" of IBM i Access Client Solutions. Consult [this document](http://www-01.ibm.com/support/docview.wss?uid=nas8N1010355) for assistance.
+### On Windows
+- Install the "Windows Application Package" of IBM i Access Client Solutions. Consult [this document](http://www-01.ibm.com/support/docview.wss?uid=nas8N1010355) for assistance.
+
+
+## Installation
+Once the prerequisites are satisfied, enter the following in the top-level directory of your LoopBack application and install this package:
+
+```
+$ npm install loopback-connector-ibmi 
+```
+
+## Configuration
+
+In LoopBack, datasources are used to store the information about your database so it can be used by the program. Use the [data source generator](https://loopback.io/doc/en/lb3/Data-source-generator.html) to add  to your application:
+
+```
+lb datasource
+```
+
+The datasource generator will then walk you through the process of setting up a datasource:
+1. `Enter the datasource name:` Any name will do, such as the name of the schema you will use or the name of your system.
+2. `Select connector for <name>:` LoopBack 3 does not have knowledge of `loopback-connector-ibmi`, so just press up on the arrow key once and select `other`.
+3. `Enter the connector's module name:` Enter `loopback-connector-ibmi`.
+4. `Install loopback-connector-ibmi`: If you haven't installed it, enter `Y`. If you have already installed it, select `n`.
+
+This will generate an entry in your `server/datasources.json` file. It should know have an entry similar to:
+
+```json
+  "test": {
+    "name": "test",
+    "connector": "loopback-connector-ibmi"
+  }
+  ```
+You should edit this entry to add information on how to connect to Db2. For `loopback-connector-ibmi`, you need to pass either a `connectionString`, or pass your `username`, `password`, and `dsn` (which will be the DSN name you set up for your ODBC driver).
```

Comment:
Ideally, the CLI prompt should ask for these common configuration properties, see my comment above.

Reply:
Noted. We'll work on this